ui/src/app.js

```diff
@@ -505,7 +505,7 @@ $(document).ready(function () {
 })
 
 function targetButton (element) {
-  var classList = element.classList
+  var classList = element.classList || []
```

Comment:
I noticed this js error when switching the language via the language menu.

btw I noticed that the button on the top left of the screen is clickable even if there are not languages to switch (only english) and the button itself is hidden.

Reply:
Cool, can you fix that, as well?

Reply:
done.
I checked the functionality with 1, 2, 3 languages and it seems working fine to me.
I also fixed the css